docs/administration/managing-users-and-teams/index.md

```diff
@@ -23,7 +23,9 @@ You should create a different User account for each person that will use Octopus
 :::
 
 ## Inviting Users {#Managingusersandteams-Invitingusers}
-
+:::hint
+Do you have an OctoID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud/index.md) to invite a user.
```

Comment:
```suggestion
Do you have an Octopus ID and a Cloud instance?  Follow [these instructions](/docs/octopus-cloud/index.md) to invite a user.
```

Reply:
Committing this suggestion so I can tweak it from a question to a directive :)